src/isomorphic/classic/class/__tests__/ReactClassMixin-test.js

```diff
@@ -368,24 +372,24 @@ describe('ReactClass-mixin', function() {
   });
 
   it('should include the mixin keys in even if their values are falsy',
-    function() {
-      var mixin = {
-        keyWithNullValue: null,
-        randomCounter: 0
-      };
+      function() {
```

Comment:
I'm leaning more and more towards a custom line-length rule (ignore length on `it` lines and docblocks). The ideal case is what Ben said where we only show the output for "changed" sections of code, perhaps we'll get there.

For now I don't think we should go out of our way to fix line length but fixing some along the way is fine.